### DISCUSSION TITLE
Problem while using 'calculate_builtin_properties'

### DISCUSSION THREAD

**Discussion by Aditya6122** [deepchecks/deepchecks#2547] [Q&A] [answered]
Getting the attribute error for the following : TextData' object has no attribute 'calculate_builtin_properties'

```
from deepchecks.nlp import TextData
from deepchecks.nlp.checks import TextPropertyOutliers

text_data = TextData(data['Title'])

# Calculate the default properties
text_data.calculate_builtin_properties()

# Run the check
TextPropertyOutliers().run(text_data)
```
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~\AppData\Local\Temp\ipykernel_5740\1170920540.py in <cell line: 7>()
      5 
      6 # Calculate the default properties
----> 7 text_data.calculate_builtin_properties()
      8 
      9 # Run the check

AttributeError: 'TextData' object has no attribute 'calculate_builtin_properties'
```

Can someone please help me with this...? Thank You

**Comment by noamzbr** [comment, marked as answer]
Hi @Aditya6122, sorry for the late response. 
Deepchecks NLP is still in beta so some things are still changing. In the stable installation this method is called `calculate_default_properties`, but it was renamed to `calculate_builtin_properties` in the latest version. I realized we have made an error in the stable version of the quickstart, in which the commented code already uses the new name... My apologies!

To resolve this, you can either use `calculate_default_properties`, or you can install the latest version directly from git (and follow the dev version of the docs). At any case we expect this issue to be resolved later this week with our next stable release for NLP.
